port_ocean/clients/port/mixins/organization.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+import httpx
+from loguru import logger
+from port_ocean.clients.port.authentication import PortAuthentication
+from port_ocean.clients.port.utils import handle_status_code
+
+
+class OrganizationClientMixin:
+    def __init__(
+        self,
+        auth: PortAuthentication,
+        client: httpx.AsyncClient,
+    ):
+        self.auth = auth
+        self.client = client
+
+    async def _get_organization_feature_toggles(self) -> httpx.Response:
+        logger.info("Fetching organization feature toggles")
```

Comment:
as you did here, seems like miss alignment with the terms, one time feature toggles and the other feature flags

Reply:
will change the python part to reflect the api schema, good point

Reply:
Fixed!